tests/run-docker

```diff
@@ -12,34 +10,14 @@ else
 fi
 
 if [ "$CMD" == "reset" ]; then
-    # Stop/Remove containers
-    docker-compose -p lizmap-${LZMBRANCH}-tests rm -sf || true
-    # Clean postgres volume
-    docker volume rm "lizmap${LZMBRANCH}_pg_data" || true
-    # Clean up lizmap config
-    ./lizmap-ctl clean
+    # Purge docker conainers/volumes
+    make reset
     exit 0
 elif [ "$CMD" == "build" ]; then
-    if [ ! -d qgis-server-plugins/lizmap ]; then
-        ./add_server_plugins.sh
-    fi
+    make build
 fi
 
-# Create a .env file so that we may use subsequent docker-compose 
-# commands directly
-cat > .env << End-of-env
-LIZMAP_USER_ID=${LIZMAP_USER_ID:-$(id -u)}
-LIZMAP_GROUP_ID=${LIZMAP_GROUP_ID:-$(id -g)}
-LZMPOSTGISVERSION=${LZMPOSTGISVERSION:-11-2.5}
-LZMQGSRVVERSION=${LZMQGSRVVERSION:-3.16}
-LZMBRANCH=${LZMBRANCH}
-LZMPGPORT=${LZMPGPORT:-8132}
-LZMQGSRVPORT=${LZMQGSRVPORT:-8131}
-LZMWEBPORT=${LZMWEBPORT:-8130}
-LIZMAP_ADMIN_LOGIN=${LIZMAP_ADMIN_LOGIN:-admin}
-LIZMAP_ADMIN_EMAIL=${LIZMAP_ADMIN_EMAIL:-admin@localhost.local}
-LIZMAP_ADMIN_DEFAULT_PASSWORD_SOURCE=${LIZMAP_ADMIN_DEFAULT_PASSWORD_SOURCE:-/srv/etc/admin.conf}
-End-of-env
+make up "$@"
```

Comment:
Why "up"?

Reply:
why not ? this is a make rule that start the stack.

Reply:
well, it does not make sens for me to do "up" when we  call "stop" (even for "build")

Reply:
> well, it does not make sens for me to do "up" when we call "stop" (even for "build")

Sorry, I do not understand the sentence.

Reply:
when we launch `./run-docker stop` (which called `docker-compose stop` before your change), now it starts containers and then stop containers. It does not make sens for me. Same thing when we do  `./run-docker build` : it starts container whereas it only built containers before the change. 

IMHO it should be only `make "@"`.

Reply:
> IMHO it should be only make "@".

Do not call make then, no point.  

`docker-run`  has a major problem to recreate env for each passed command, which is a pain when if one wants to customize environnement, and we should be able:
1. recreate the environment only without running any command
2. Use docker-compose commants directly without recreating the environment.

 If you stlill want to pass command directly to docker-compose in `./run-docker then just replace `make up` with `docker-compose $@`